### ISSUE TITLE
Another colorblind accessibility issue

### ISSUE THREAD

**Issue by jmandel** [zulip/zulip-mobile#3531]
Similar to #3524 I can't read the highlighted text in:
![Screenshot_20190618-225014](https://user-images.githubusercontent.com/313089/59735557-ab773780-921b-11e9-9fb7-562c8226f661.png)

Incidentally, I'm not sure why this particular word is highlighted or what the highlighting indicates.

**Comment by ishammahajan** [comment]
@jmandel thanks for reporting this! Is it perhaps that you're using the 'Alert Words' feature? For reference for the other devs, here's what it looks like in the WebApp.

<img width="381" alt="Screenshot 2019-06-19 at 1 26 50 PM" src="https://user-images.githubusercontent.com/33200024/59747165-f46bd200-9295-11e9-8893-08a74b06d9f2.png">

As it goes, I think the highlight on the mobile app is very harsh, and should be toned down.

**Comment by jmandel** [comment]
Yes, good call. It's an alert word - and indeed, I couldn't read it :)

**Comment by gnprice** [comment]
Thanks @jmandel for the report!

@ishammahajan , if you'd like to claim this and take it the rest of the way, I think it's another one you're well equipped to take care of :slightly_smiling_face: 

The highlighting in the webapp (in Isham's screenshot above, in night mode) doesn't look great to me either. @jmandel in the same message you have above, can you read the word well in the webapp? If so then we should certainly copy the webapp here as an immediate accessibility fix, but either way it's possible it'd be good to then adjust the design on both platforms, at least in night mode.

**Comment by gnprice** [comment]
I just edited the title to reflect what we figured out above about what this feature is :slightly_smiling_face: 

@jmandel , if you'd be up for checking how this works for you in day/light mode, that'd be helpful too.

**Comment by jmandel** [comment]
Thanks! In light mode, it looks fine for me.

**Comment by jmandel** [comment]
Also, to be complete: In dark mode the mobile app and the web app are both unreadable for me.

**Comment by gnprice** [comment]
> Also, to be complete: In dark mode the mobile app and the web app are both unreadable for me.

Ah, OK! I'll go turn this into an issue on zulip/zulip, then, for a bit of a wider audience -- and whatever we end up with, we can implement on both web and mobile.

**Comment by ishammahajan** [comment]
> @ishammahajan , if you'd like to claim this and take it the rest of the way, I think it's another one you're well equipped to take care of 🙂

Sure!

> Ah, OK! I'll go turn this into an issue on zulip/zulip, then, for a bit of a wider audience -- and whatever we end up with, we can implement on both web and mobile.

Once a consensus is reached there, I think I'll pick this up 😄 .

**Comment by ishammahajan** [comment]
So I checked the color used in the webapp to highlight an alert-word, and found that it is `hsla(102, 85%, 57%, 0.5)`, which is exactly the color used in our mobile app.

<img width="344" alt="Screenshot 2019-06-24 at 11 47 02 AM" src="https://user-images.githubusercontent.com/33200024/59995794-0f638b00-9676-11e9-9f11-a11d298fe3dc.png">

Looks like we're using this already (looks exactly like the webapp).

<img width="598" alt="Screenshot 2019-06-24 at 11 51 56 AM" src="https://user-images.githubusercontent.com/33200024/59995941-7da84d80-9676-11e9-9802-fc2e2e0a519a.png">

**Comment by ishammahajan** [comment]
So the above commit doesn't fix the issue completely, it just applies sort of a temporary patch to make things more readable than before.

> Also, to be complete: In dark mode the mobile app and the web app are both unreadable for me.

@jmandel, would you please read the following and confirm if reducing the alpha (increasing the transparency) helped at all?
<img width="466" alt="Screenshot 2019-06-24 at 5 27 14 PM" src="https://user-images.githubusercontent.com/33200024/60016973-51f08c00-96a5-11e9-90bd-4f6e03d264e9.png">

**Comment by jmandel** [comment]
Thanks! This is an improvement but still kind of difficult to read. I would
go farther if possible.

**Comment by gnprice** [comment]
Just merged that alpha adjustment #3538.

Tracking the need for a better fix over at zulip/zulip#12614 (which I filed earlier, as mentioned in this thread). Looks like we haven't found someone to take that up yet.
